livekit-plugins/livekit-plugins-deepgram/livekit/plugins/deepgram/stt.py

```diff
@@ -311,8 +311,7 @@ def _end_speech(self) -> None:
             )
             return
 
-        if len(self._final_events) == 0:
```

Comment:
This is actually a normal case. deepgram SpeechStarted (and VAD generally) does not mean it found transcribable audio